src/asmcnc/apps/drywall_cutter_app/config/config_loader.py

```diff
@@ -11,8 +12,10 @@
 def debug_decorator(func):
     def wrapper(*args, **kwargs):
         if DEBUG_MODE:
-            print('Calling function: ' + func.__name__ + ' with args: ' + str(args[-1:]) + ' and kwargs: ' + str(kwargs))
```

Comment:
When calling static function the [-1:] was removing first arg (which wasn't self in this case).